x-pack/plugins/apm/public/components/shared/DatePicker/__test__/DatePicker.test.tsx

```diff
@@ -64,35 +64,33 @@ describe('DatePicker', () => {
   });
 
   beforeEach(() => {
-    jest.clearAllMocks();
+    jest.resetAllMocks();
```

Comment:
clearAllMocks only clears the calls, not the state. Some of the state from previous tests was leaking into others.